go.mod

```diff
@@ -86,7 +86,7 @@ replace (
 	k8s.io/kube-proxy => k8s.io/kube-proxy v0.24.6
 	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.24.6
 	k8s.io/kubectl => k8s.io/kubectl v0.24.6
-	k8s.io/kubelet => github.com/kubewharf/kubelet v1.24.6-kubewharf.5
+	k8s.io/kubelet => github.com/WangZzzhe/kubelet v0.0.0-20230921082135-659f02530842
```

Comment:
what is new for this kubelet version?

Reply:
Const NumericTopologyManagerPolicy is defined. It is used to generate node topology policy in CNR for topology match schedule plugin.  https://github.com/kubewharf/katalyst-api/pull/39/files#diff-22e95ad850935a2edd7e57a7ac7db25dbe67e516e0d7a20656184b6b4290f3d7R32-R33

Reply:
will this be merged into kubewharf-versioned kubelet?

Reply:
yes, go.mod should be updated after [https://github.com/kubewharf/enhanced-k8s/pull/12](url) merged

Reply:
@WangZzzhe The enhanced-k8s PR has been merged. Could you rebase it?

Reply:
> @WangZzzhe The enhanced-k8s PR has been merged. Could you rebase it?

Done.